nomad/structs/structs.go

```diff
@@ -8504,6 +8508,9 @@ func (e *Evaluation) NextRollingEval(wait time.Duration) *Evaluation {
 		Status:         EvalStatusPending,
 		Wait:           wait,
 		PreviousEval:   e.ID,
+		CreateTime:     now,
+		ModifyTime:     now,
+		// TODO(@jasmine): is a NextRollingEval technically created now or when original eval was created?
```

Comment:
Get rid of this comment.

Reply:
> is a NextRollingEval technically created now or when original eval was created?

But this question still stands. Should CreateTime use `now` or `e.CreateTime`? The philosophical question I'm really asking is: is a NextRollingEval and brand new eval? Same question goes for CreateBlockedEval and CreateFailedFollowUpEval

Reply:
Answered here https://github.com/hashicorp/nomad/pull/5881#discussion_r311224050
> The new eval will reference the old eval but they are distinct evals and therefore should have distinct created times.

Reply:
yes nextRollingEval and CreateBlockedEval/CreateFailedFollowupEval are all new evals and should be using `now` so your comment can be deletd.